plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -94,6 +95,7 @@ export class EventPipelineRunner {
     }
 
     async runEventPipeline(event: PipelineEvent): Promise<EventPipelineResult> {
+        this.originalEvent = event
```

Comment:
It's confusing that we store it in the constructor and also here. Does just 1 or the other not work?

Reply:
right, forgot to remove it from here ...  I added if before creating a function outside of the class that call locations can use